runtime/src/test/resources/flux_config.yml

```diff
@@ -0,0 +1,7 @@
+#guiceModuleClass: "com.flipkart.flux.examples.externalevents.SellerVerificationWorkflowModule"
+workflowClasses: ["com.flipkart.flux.dao.TestWorkflow",
+                  "com.flipkart.flux.integration.SimpleWorkflow"]
+taskConfig:
+  com.flipkart.flux.integration.SimpleWorkflow_simpleStringReturningTask:
+    execution:
```

Comment:
I think "execution_concurrency" would be good, as there won't be many properties under execution.